nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -647,8 +666,9 @@ class BaseSearchRequest(BaseModel):
         title="Filters",
         description="The list of filters to apply. Filtering examples can be found here: https://docs.nuclia.dev/docs/rag/advanced/search/#filters",  # noqa: E501
     )
-    page_number: int = SearchParamDefaults.page_number.to_pydantic_field()
+    page_number: int = SearchParamDefaults.page_number.to_pydantic_field(deprecated=True)
     page_size: int = SearchParamDefaults.page_size.to_pydantic_field()
```

Comment:
this one should be deprecated too no?

Reply:
They are deprecated in the field itself, but I can add it for readability